content/attacks/posts/2024-02-23-BitForex.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-02-23
+target-entities: BitForex
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Exit Scam
+  - Wallet Hack
+title: "BitForex's Suspicious Outflows: Hack or Exit Scam?"
+loss: 56500000
+---
+
+## Summary
+
+Since February 23, 2024, BitForex, a crypto exchange operational since 2017, ceased processing withdrawals amidst unexplained outflows of about $56.5M worth of crypto from its hot wallets. The absence of communication from BitForex, coupled with the recent departure of its CEO Jason Luo, has raised concerns over a potential inside job or exit scam. 
+
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
```

Comment:
You already mentioned CEO of BitForex, you can mention him here. Preferably with sources with official accusations.